dialog/src/main/java/roboy/dialog/states/devoStates/QuestionRoboyQAState.java

```diff
@@ -268,61 +227,43 @@ private String inferPasAnswer(Map<SemanticRole, String> pas, Roboy roboy) {
+            if (matchPas(pas, new Pair(SemanticRole.PATIENT, "you"))) {
+                answer = extractNodeNameForPredicate(Neo4jProperty.full_name, roboy);
             } else if (matchPas(pas, new Pair(SemanticRole.PATIENT, ".*\\b(father|dad)\\b.*"))) {
                 answer = extractNodeNameForPredicate(Neo4jRelationship.CHILD_OF, roboy);
             } else if (matchPas(pas, new Pair(SemanticRole.PATIENT, ".*\\b(sibling|brother)\\b.*"))){
                 answer = extractNodeNameForPredicate(Neo4jRelationship.SIBLING_OF, roboy);
-            } else if (matchPas(pas, new Pair(SemanticRole.PREDICATE, "created|made|built"), new Pair(SemanticRole.PATIENT, "you"))) {
-                answer = extractNodeNameForPredicate(Neo4jRelationship.CREATED_BY, roboy);
             }
         } else if (matchPas(pas, new Pair(SemanticRole.PREDICATE, "do|like"), new Pair(SemanticRole.AGENT, "you"))){
-//            double prob = Math.random();
-//            if (prob < .3) {
-//                answer = extractNodeNameForPredicate(Neo4jProperty.abilities, roboy);
-//            } else if(prob < .7) {
-//                answer = extractNodeNameForPredicate(Neo4jRelationship.HAS_HOBBY, roboy);
-//            } else {
+            double prob = Math.random();
+            if (prob < .3) {
+                answer = extractNodeNameForPredicate(Neo4jProperty.abilities, roboy);
+            } else if(prob < .7) {
+                answer = extractNodeNameForPredicate(Neo4jRelationship.HAS_HOBBY, roboy);
+            } else {
                 answer = extractNodeNameForPredicate(Neo4jProperty.skills, roboy);
-//            }
-        } else if (matchPas(pas, new Pair(SemanticRole.PREDICATE, "dream|wish"), new Pair(SemanticRole.AGENT, "you"))) {
-            answer = extractNodeNameForPredicate(Neo4jProperty.dreams, roboy);
-        }
-        else if (matchPas(pas, new Pair(SemanticRole.PATIENT, "you"), new Pair(SemanticRole.PREDICATE, "are"),
-                new Pair(SemanticRole.AGENT, "what"))) {
-            answer = extractNodeNameForPredicate(Neo4jProperty.name, roboy) + " "
-                    + extractNodeNameForPredicate(Neo4jProperty.identitiy, roboy);
+            }
         }
 
         return answer;
     }
 
     private String inferTripleAnswer(List<Triple> triples, Roboy roboy) {
         String answer = "";
-        for (Triple t: triples) {
-            if (t.subject.matches(".*\\b(meet|see|know)\\b.*")) {
-                answer = infoValues.getSuccessAnswers(Neo4jProperty.media).getRandomElement();
-            }
-        }
         // else if {OBJ: *} -> query * -> I'm sure I know a typeof(*) called *! (Where does he live? :))
         //
         // 	if * in Neo4jRelationship.FRIEND_OF
         return answer;
     }
 
     private Neo4jRelationship inferPredicateFromObjectAnswer(String objectAnswer) {
-        if (objectAnswer.contains("hobby")) {
+        if (objectAnswer.contains("hobb")) {
```

Comment:
hobby

Reply:
IIRC @josephbirkner mentionned, this was so that both hobby and hobbies are accepted